### PR TITLE
Add notes about icons for xwayland apps

### PR DESCRIPTION
**Pull request by joshuataylor**

### CODE REVIEW

File: doc/sfwbar.rst [hunk 1]
```diff
@@ -707,6 +707,30 @@ tricky). Mapping is supported via top-level ``MapAppId`` keyword. I.e. ::
 where app_id is the desired app_id and pattern is a regular expression to
 match the title against.
 
+If you are using an XWayland app, they usually do not have an `app_id` set. If
+an icon is not showing, you can add your icon to the following locations:
+1. `$HOME/.icons`
```
Comment (LBCrion) [inline doc/sfwbar.rst:712]: sfwbar will also look for icons in:
1) location of the main config file currently in use
2) $XDG_CONFIG_HOME/sfwbar/
3) One of directories listed in $XDG_DATA_DIRS/sfwbar/

File: doc/sfwbar.rst [hunk 2]
```diff
@@ -707,6 +707,30 @@ tricky). Mapping is supported via top-level ``MapAppId`` keyword. I.e. ::
...
+2. One of the directories listed in `$XDG_DATA_DIRS/icons`
+3. /usr/share/pixmaps
+
+If an `app_id` is not set, sfwbar will fallback to using the `instance` in the
+`window-properties`, which can be found by using a tool such as `swaymsg` (if on sway)
```
Comment (LBCrion) [inline doc/sfwbar.rst:717]: We should probably say that users can check programs app_id with Hyprland using 'hyprctl -j clients' and for other compositors supporting wlr-foreign-toplevel protocol (i.e. labwc, wayfire) using 'sfwbar -d -g app_id`

Comment (LBCrion) [reply doc/sfwbar.rst:717]: I also just made a change in the latest git version to support 'sfwbar -d -g app_id' with all IPCs. This should make icon lookup easier across different compositors.

**Comment by joshuataylor** [comment]
Thanks for the notes, I've added more documentation around the new command (awesome!!).

**Pull request merged**